HACKING.md

```diff
@@ -67,10 +77,16 @@ helper classes.
 
 #### `flutter.py`
 
+Tool to verify and mark validity for any Giza-style YAML input.
+
 #### `types.py`
 
+Definitions of class components that make up the parser. See n.py for more granular types of nodes that are expected within the reStructuredText.
```

Comment:
I actually think `n.py` is worth having its own section!

Reply:
done!